olm/check-yq.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+if ! which yq &> /dev/null
+then
+  echo
+  echo "#### ERROR ####"
+  echo "####"
+  echo "#### Please install the 'yq' tool before being able to use this script"
+  echo "#### see https://github.com/kislyuk/yq"
+  echo "#### and https://stedolan.github.io/jq/download"
+  echo "####"
```

Comment:
Could just tell people to use `/usr/bin/pip2.7 install yq jq` but I guess OS flavours vary (brew, pip, etc.)